config/lib/agenda.js

```diff
@@ -0,0 +1,9 @@
+var Agenda = require('agenda'),
```

Comment:
I looked into switching [eslint strict](http://eslint.org/docs/rules/strict) rule on because of this, but got too many warnings and backed up. ;-) Seems like we're using it in quite mixed fashion, which complicates linting.